FileParser.js

```diff
@@ -253,7 +254,24 @@ var FileParser = (function ()
 			});
 		}
 	};
+	FileParser.prototype.processVariableLine = function(line)
+	{
+		// var VarName = (line.substring(0,line.indexOf("="))).trim();
+		// //there is a bug with the re-assignment of variables here. I couldn't find it!
+		// var VarValue = (line.substring(line.indexOf("=")+1)).trim();
+    	var parts = line.split('=', 2);
+   	 	var varName = parts[0].replace("$","");
```

Comment:
I thought you decided not to replace the `$` character? You'll have to add it back in here if that's the case:

    line = line.replace(new RegExp("\\"+Vars,'g'), this.CustomVariables[Vars]);